controllers/articles.controller.js

```diff
@@ -15,3 +15,17 @@ exports.getArticles = (req, res, next) => {
         .catch(err => next(err));
 
 }
+
+exports.patchVotesByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+    let update = req.body;
+
+    //if input an empty object send back orginal article
+    if(Object.keys(update).length === 0) {
+        update = { inc_votes : 0 };
+    } 
```

Comment:
Do we need to handle this eventuality? What will happen if votes doesn't exist on the body?

Reply:
Hi, I wanted to return a bad request. I was trying to send a 200